tests/functional/fx_fennec_v1_sign_up.js

```diff
@@ -64,11 +64,11 @@ define([
         .end()
 
         // uncheck the passwords and history engines
-        .findByCssSelector('input[value="passwords"]')
+        .findByCssSelector('div.two-col-block:nth-child(2) > div:nth-child(1) > label:nth-child(1)')
           .click()
         .end()
 
-        .findByCssSelector('input[value="history"]')
+        .findByCssSelector('div.two-col-block:nth-child(2) > div:nth-child(2) > label:nth-child(1)')
```

Comment:
Would it be easier to give the `<label>` tags a unique id or classname to help with the [arguably complex] `:nth-child(x)` logic?